src/lib/templateengine.js

```diff
@@ -1321,6 +1336,8 @@ function TemplateEngine( undefined ) {
     
     if (page_id.match(/^id_[0-9_]*$/) == null) {
       // find Page-ID by name
+      // decode html code (e.g. like &apos; => ')
+      page_id = $("<textarea/>").html(page_id).val();
```

Comment:
I'm not too fond of using JQuery (carelessly) anymore as it's somtimes quite slow... It's good to get code written quickly - but not for quick code ;)

In this specific case: do we need such an universal solution? Or would a more specific escape / mapping also work?